opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -364,7 +364,7 @@ def create_span(
         span_id = generate_span_id()
         if parent is Tracer.CURRENT_SPAN:
             parent = self.get_current_span()
-        if parent is None:
+        if parent in {None, trace_api.INVALID_SPAN_CONTEXT}:
```

Comment:
Good questions. 
My understanding is that `INVALID_SPAN_CONTEXT` is a special object, in fact I have some concerns because the B3 propagator returns `SpanContext(0,0)` and not `INVALID_SPAN_CONTEXT`: https://github.com/open-telemetry/opentelemetry-python/issues/233#issuecomment-545607680.

I think `parent is None or not parent.is_valid()` would work just fine. Maybe a little bit slower but more flexible as propagators don't have to return `INVALID_SPAN_CONTEXT`.

Reply:
Shouldn't `parent` be assigned to `None` when it is `INVALID_SPAN_CONTEXT` (is not valid in general)`?
If we don't do that other components like exporters will have to check also if parent is valid or not to understand if the span is a root one.

```
...
if not parent.is_valid():
   parent = None
if parent is None:
....
```

Does it make sense?

Reply:
I think is_valid is a good approach. I figure INVALID_SPAN_CONTEXT should be something that people can check against, but at the same time there's nothing stopping people from generating span contexts using invalid span / trace ids.

I guess a bigger risk there is that one can load invalid span and trace ids from header values, and those would generate an invalid span context that is not the same object as INVALID_SPAN_CONTEXT.

> Shouldn't `parent` be assigned to `None` when it is `INVALID_SPAN_CONTEXT` (is not valid in general)`?
> If we don't do that other components like exporters will have to check also if parent is valid or not to understand if the span is a root one.
> 
> ```
> ...
> if not parent.is_valid():
>    parent = None
> if parent is None:
> ....
> ```
> 
> Does it make sense?

Agreed, will do that.